tests/publiccloud/slem_prepare.pm

```diff
@@ -26,6 +26,24 @@ sub run {
         trup_call("pkg install $runtime toolbox");
         $instance->softreboot();
     }
+    if (get_var("PUBLIC_CLOUD_SELINUX")) {
+        # Enable SELinux
+        record_info("SELinux", "Enabling SELinux on this instance");
+        trup_call("pkg in policycoreutils-python-utils");
+        $instance->softreboot();
+        # This adds 'security=selinux selinux=1 enforcing=1' to GRUB_CMDLINE_LINUX_DEFAULT in /etc/default/grub
+        assert_script_run('sed \'s/^\(GRUB_CMDLINE_LINUX_DEFAULT="\)\(.*\)\("\)$/\1\2 security=selinux selinux=1 enforcing=1"/g\' -i /etc/default/grub');
```

Comment:
SLEM uses the following command to enable selinux:
`transactional-update setup-selinux`
I would prefer to use the official way so we can also test this command which comes as part of the product.